code-for-canada/project-thundercat
# [Functionality] Building Submit Test Footer

# Description

I created a new generic component called '_TestFooter.jsx_', so we can add this footer in all the desired tests.

The **Submit Test** button still works as expected, but the **Quit Test** button doesn't have any action/behavior for now. I'll create another PR with a new generic popup window component and add an action to this **Quit Test** button.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Instruction Tab [EN]:**

![image](https://user-images.githubusercontent.com/23021242/53955542-44d47080-40a7-11e9-9211-7dc683f681b5.png)

**Instruction Tab [FR]:**

![image](https://user-images.githubusercontent.com/23021242/53955505-2cfcec80-40a7-11e9-9705-03a10cba17a3.png)

**Background Tab [EN]:**

![image](https://user-images.githubusercontent.com/23021242/53955424-fb842100-40a6-11e9-88d2-eb0782504dfe.png)

**Background Tab [FR]:**

![image](https://user-images.githubusercontent.com/23021242/53955439-0343c580-40a7-11e9-87bd-61422fcecfb3.png)

**Inbox Tab [EN]:**

![image](https://user-images.githubusercontent.com/23021242/53955570-60d81200-40a7-11e9-8993-4767fed18ade.png)

**Inbox Tab [FR]:**

![image](https://user-images.githubusercontent.com/23021242/53955586-67ff2000-40a7-11e9-8157-a25c5d0fa3c1.png)

# Testing

Manual steps to reproduce this functionality:

1.  Go to _Prototype_ tab.
2.  Click _Start eMIB Sample Test_.
3.  Navigate in the test and make sure that the **Submit Test** button still works as expected.
4. Navigate in all browser to make sure everything looks great as well.

Screenshot of unit tests run passing:

![image](https://user-images.githubusercontent.com/23021242/53960731-06de4900-40b5-11e9-8f91-0c8d6dfe1b58.png)

# Checklist

Applicable for all code changes.
